utils/mcmb/mcmb.c

```diff
@@ -409,23 +392,20 @@ static struct cmb_xitem *cmb_transform_find;
     for (n = 1; n < nitems; n++)                                              \
       {                                                                       \
         (void)memcpy(&xitem, &items[n], sizeof ( struct cmb_xitem * ));       \
-        if (!opt_silent && !opt_quiet)                                        \
+        if (!opt_quiet)                                                       \
```

Comment:
trailing whitespace?

Reply:
It's OK—it's inside a macro where the continuation backslash is right-justified.